ale/drivers/mex_drivers.py

```diff
@@ -336,8 +326,17 @@ def line_scan_rate(self):
         : list
           Line scan rates
         """
-        times = [time - self.center_ephemeris_time for time in self.binary_ephemeris_times]
-        return (self.binary_lines, times, self.binary_exposure_durations)
+        relative_times = [time - self.center_ephemeris_time for time in self.binary_ephemeris_times]
```

Comment:
Suggest adding documentation on what and why this is so much more than the simple case. Reading the code without the knowledge of why (unless it is obvious) forces the next programmer to go looking for documentation before they can decide if it is working as intended.

Reply:
Agreed that some documentation would be good here!